src/components/mapComponents/MapTiles.tsx

```diff
@@ -23,25 +23,15 @@ const MapTiles: FC<MapTilesProps> = ({
   addReportMarker,
   isCreateReportMapMode,
 }) => {
-  const color = ({approvedBySupervisor, approvedByEditor, usersCount}: Tile): string => {
-    if (approvedBySupervisor) {
-      return colors.colorApprovedBySupervisor;
-    }
+  const color = ({approvedByEditor}: Tile): string => {
```

Comment:
could be used as ternary operation instead of function as well

Reply:
I've replaced body with ternary operator, but it has to stay as function I believe, because it needs a parameter